src/IList.js

```diff
@@ -0,0 +1,14 @@
+function IList() {}
+
+IList.prototype.clear = function() {console.log('IList cleat'); }
+IList.prototype.toString = function() {console.log('IList toString');}
+IList.prototype.getSize = function() {console.log('IList getSize');}
+IList.prototype.push = function() {console.log('IList push');}
```

Comment:
`IList.prototype.push = function(value) {console.log('IList push');}`
No arguments for methods. Why?